# Calculate format=diff pkg(sys-apps/calculate-utils)<3.4.1
Поддержка названия gentoo для /usr/portage

--- /usr/lib/python2.7/site-packages/calculate/builder/builder.py
+++ /usr/lib/python2.7/site-packages/calculate/builder/builder.py
@@ -149,7 +149,7 @@ class Builder(Update):
                         self.clVars.Get('update.cl_update_layman_make'),
                         self.clVars.Get('update.cl_update_layman_conf'),
                         prefix=self.clVars.Get('cl_builder_path'))
-        if repname != "portage":
+        if repname not in ("portage", "gentoo"):
             layman.remove(repname, rpath_orig)
             removeDir(rpath)
         return True
--- /usr/lib/python2.7/site-packages/calculate/update/update.py
+++ /usr/lib/python2.7/site-packages/calculate/update/update.py
@@ -293,7 +293,7 @@ class Update(object):
                                 dv.Get('cl_update_layman_make'),
                                 dv.Get('cl_update_layman_conf'),
                                 prefix=chroot_path)
-                if repname != "portage":
+                if repname not in ("portage", "gentoo"):
                     layman.add(repname, url, rpath_orig)
                 if not self._syncRepository(repname, url, rpath, revision,
                                      cb_progress=self.setProgress):
@@ -328,7 +328,7 @@ class Update(object):
                         dv.Get('cl_update_layman_make'),
                         dv.Get('cl_update_layman_conf'),
                         prefix=chroot_path)
-        if repname != "portage":
+        if repname not in ("portage", "gentoo"):
             # TODO: debug block
             #print "EEEE",repname, url, rpath_orig
             layman.add(repname, url, rpath_orig)
@@ -340,7 +340,7 @@ class Update(object):
         """
         Спрятать кэш
         """
-        if name in ("portage",):
+        if name in ("portage", "gentoo"):
             return
         if not name in OverlayOwnCache(self.clVars):
             for cachename in self.metadata_cache_names:
@@ -360,7 +360,7 @@ class Update(object):
         """
         Извлеч кэш
         """
-        if name in ("portage",):
+        if name in ("portage", "gentoo"):
             return
         cachenames = self.metadata_cache_names
         if not name in OverlayOwnCache(self.clVars):
